README.md

```diff
@@ -10,7 +10,10 @@ Obelisk provides an easy way to develop and deploy your [Reflex](https://github.
 - [Developing an Obelisk project](#developing-an-obelisk-project)
   - [Adding Packages](#adding-packages)
   - [Adding Package Overrides](#adding-package-overrides)
-  - [Running over https](#running-over-https)
+  - [Running over HTTPS](#running-over-https)
+  - [IDE Support](#ide-support)
+    - [Terminal-based feedback](#terminal-based-feedback)
+    - [**Experimental** `ghcide` support](#experimental-ghcide-support)
```

Comment:
This shouldn't be bold, it makes it stand out more than other items

Reply:
I'll fix. Thanks.